alsrb0504/20230509/테이블 해시 함수/테이블 해시 함수.js

```diff
@@ -0,0 +1,46 @@
+function solution(data, col, row_begin, row_end) {
+  col--;
+  row_begin--;
+  row_end--;
+
+  // 문제에서 주어진 조건으로 정렬
+  data.sort((a, b) => {
+    if (a[col] === b[col]) return b[0] - a[0];
+
+    return a[col] - b[col];
```

Comment:
요긴 삼항연산자를 이용해서 간결하게 표현할 수 있을 것 같아요!
```suggestion
    a[col] === b[col] ? return b[0] - a[0] : a[col] - b[col]
```